docs/Troubleshooting.md

```diff
@@ -21,3 +21,22 @@ You can kill the process by:
 
   * on Windows OS: run `taskkill /F /PID ...` all instances
   * on other OS: run `kill -9 ...` all instances
+
+### The Language Server Crashes Due to an Out Of Memory Error
+
+If you are working with large XML files or referencing large schema files,
+this may lead to the language server running out of memory.
+The Java language server is more likely to run out memory than the binary language server.
+Switching to the binary language server
+or increasing the memory available to the Java language server could resolve this issue.
+
+If you get an Out of Memory Error, but aren't working with large XML files,
+then there may be a memory leak in the language server.
+Please [file a issue](https://github.com/redhat-developer/vscode-xml/issues) with a description of what you were doing if this is the case.
+
+#### How to increase the amount of memory available to the Java Language Server
+
+1. Go to settings
+2. Navigate to the setting `xml.server.vmargs`
+3. Add `-Xmx 1.5G` to the setting string. This allows the the language server to use at most 1.5 gigabytes of memory.

```

Comment:
there's no space between -Xmx and the value

-Xmx512m would probably be good enough. We're starting from 64M https://github.com/redhat-developer/vscode-xml/blob/master/package.json#L122. 1.5G is a huge step forward

Reply:
My mistake. I think I was looking at vscode-java :| . Also, yes, I figured maybe it was fine with the space, but looks like it may not get set this way.